packages/components/psammead-navigation/README.md

```diff
@@ -6,7 +6,7 @@ This component is currently tagged as alpha and is not suitable for production u
 
 ## Description
 
-The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl` and `NavigationLi`. They use `ul` and `li` HTML elements respectively.
+The `@bbc/psammead-navigation` package is a set of two components, `NavigationUl` and `NavigationLi`. They use `ul` and `li` HTML elements respectively. The package also includes a Hamburger Menu and a Dropdown Navigation which are visible for breakpoints under 600px.
```

Comment:
You can also add a `Exports` section so that it becomes clear on how `@bbc/psammead-navigation` includes the `Hamburger Menu` and `Dropdown Navigation`
### Exports
- `/Dropdown` - Which is visible for breakpoints under 600px